src/mmw/js/src/modeling/models.js

```diff
@@ -71,13 +71,14 @@ var ProjectModel = Backbone.Model.extend({
 
     defaults: {
         name: '',
-        created_at: null,          // Date
-        area_of_interest: null,    // GeoJSON
-        model_package: '',         // Package name
-        scenarios: null,           // ScenariosCollection
-        user_id: 0,                // User that created the project
-        is_activity: false,        // Project that persists across routes
-        needs_reset: false         // Should we overwrite project data on next save?
+        created_at: null,           // Date
+        area_of_interest: null,     // GeoJSON
+        model_package: '',          // Package name
+        scenarios: null,            // ScenariosCollection
+        user_id: 0,                 // User that created the project
+        is_activity: false,         // Project that persists across routes
+        needs_reset: false,         // Should we overwrite project data on next save?,
+        area_of_interest_name: null // Human readable string for AOI.
```

Comment:
Since the ordering of these fields doesn't matter, can we move `area_of_interest_name` right below `area_of_interest` so they are close together? I also liked that `needs_reset`, which doesn't have a corresponding database field, was at the very end, after `is_activity` a related field.